src/content_script.js

```diff
@@ -66,6 +66,14 @@ const SHARE_PATTERN_DETECTION_SELECTORS = [
   "[title='Share on Facebook']" // Medium
 ];
 
+// Selects all anchor links with an HREF to Facebook Domains
+const NOREFERRER_DETECTION = [
+  "a[href*='facebook.com']",
+  "a[href*='instagram.com']",
+  "a[href*='messenger.com']",
+  "a[href*='workplace.com']"
+];
```

Comment:
These are some extremely broad and pervasive selectors. We should get a performance consult here to see if this is the best/only way to do this.

Reply:
e.g., is it more performant to do a faster selector to get all the `a[href]` elements first, and then only using the value-seeking `*=` selector on that smaller set?